pkgs/servers/dns/powerdns/default.nix

```diff
@@ -15,12 +15,7 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ pkgconfig ];
   buildInputs = [
     boost mysql57.connector-c postgresql lua openldap sqlite protobuf geoip
-    libyamlcpp libsodium curl opendbx unixODBC botan2
-  ];
-
-  patches = [
-    # checksum type not found, maybe a dependency is to old?
-    ./skip-sha384-test.patch
+    libyamlcpp libsodium curl opendbx unixODBC botan2 libressl
```

Comment:
@ajs124 does this explicitly need to be `libressl`, or is `openssl` ok too?

Reply:
See the (extended) commit message: "neither openssl 1.0 (headers not found) nor openssl 1.1 (configuration failed when trying to test constants) work, but libressl does"

So from my experiments, yes, for some reason it does.